client/me/help/help-courses/course.jsx

```diff
@@ -0,0 +1,24 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { localize } from 'i18n-calypso';
+import Card from 'components/card';
+
+export class Course extends Component {
```

Comment:
As suggested [here](https://github.com/Automattic/wp-calypso/pull/7595#discussion_r75828364), I think we don't want to export un-localized component here, too.